extensions/interactions/base_test.py

```diff
@@ -105,11 +111,22 @@ def _validate_customization_arg_specs(self, customization_args):
                     obj_class.normalize(ca_spec['default_value']))
 
     def _validate_dependencies(self, dependency_ids):
+        """Validates all the dependencies given their dependency ids.
```

Comment:
I think only the IDs are validated right?
```suggestion
        """Validates all the dependency ids given.
```

Also, should we remove the single line comment saying basically the same thing?

Reply:
Done